hcl2template/types.packer_config.go

```diff
@@ -639,6 +642,47 @@ func (cfg *PackerConfig) getCoreBuildPostProcessors(source SourceUseBlock, block
 	return res, diags
 }
 
+// GetHCPPackerRegistryBlock return the HCP registry configuration block
+// that can should be used for the current build. Right now, it should
+// use the block at the top level but support the block inside the first
+// build block with a deprecation diagnostic
```

Comment:
Maybe I missed it, but there doesn't seem to be a deprecation diagnostic returned anywhere in this function?

Reply:
Looking at the tests quickly, it seems that intuition is correct, I haven't seen a test that produces a diagnostic that isn't an error, so the deprecation warning is never produced I'd think